docs/source/multimodal/speech_llm/intro.rst

```diff
@@ -21,19 +23,20 @@ Model Architecture
 ^^^^^^^^^^^^^^^^^^
 
 One way to incorporate speech into LLM is to concatenate speech features with the token embeddings of the input text prompt before being fed into the LLM. In this way, the LLM can have direct access to the speech information when generating the output text.
```

Comment:
revise.

One way to incorporate speech into an LLM is to concatenate speech features with the token embeddings of the input text prompt before feeding them into the LLM.